Day04/Module11UnderstandingSequelize/index.js

```diff
@@ -0,0 +1,188 @@
+const express = require('express');
+const app = express();
+
+const employeeRoutes = require('./routes/employeeRoutes')
+
+const sequelise = require('./Model/DatabseSequelize');
+
+const Employee = require('./Model/EmployeeModel');
+const manager = require('./Model/managerModel');
+
+const bodyparse = require('body-parser');
+
+app.use(bodyparse.urlencoded({extended:true}));
+
+// to convert body data into json
+app.use(bodyparse.json());
+
+//app.get('/employee/form',employeeRoutes);
+
+app.get('/',(req,res,next)=>{
+     res.send(`<h1>please use correct URL</h1>`);
+});
+
+app.get('/add',(req,res,next)=>{
+     res.send(`<h2>Submit Form</h2>
+
+     <form action="/employee/add" method="post">
+     <label for="name">Name:</label><br>
+     <input type="text" id="name" name="name"><br>
+     
+     <label for="company">Company:</label><br>
+     <input type="text" id="company" name="company"><br>
+     
+     <label for="location">Location:</label><br>
+     <input type="text" id="location" name="location"><br><br>
+     
+     <input type="submit" value="Submit">
+   </form>`)
+});
+
+
+
+
+app.get('/update',(req,res,next)=>{
+    res.send(`<h2>Submit Form</h2>
+
+    <form action="/employee/update" method="post">
+    <label for="name">Name:</label><br>
+    <input type="text" id="name" name="name"><br>
+    
+    <label for="company">Company:</label><br>
+    <input type="text" id="company" name="company"><br>
+    
+    <label for="location">Location:</label><br>
+    <input type="text" id="location" name="location"><br><br>
+    
+    <input type="submit" value="Submit">
+  </form>`)
+})
+
+
+
+
+
+
+
+// Inserting data in table using sequelize 
+// alway use promises 
+app.get('/employee/show',(req,res,next)=>{
+     Employee.findAll().then((resolve)=>{
+        console.log('get all the data')
+        for(const employee of resolve){
+             console.log(employee);
+        }
+        res.send('<h2>We have get all the details<h2/>')
```

Comment:
Why we are not sending employee data in the response?
By this logic, we are logging the employee data and sending text message that we got it

Have to handle the exception as well with try and catch